### PR TITLE
added dynamic altroom portals tweens generation;

### PR DESCRIPTION
**Pull request by TeslaRus**
now in every success  rooms flipping altroom portal tweens are rebuild-ed;
no more hacky logic: correct in all cases;

### CODE REVIEW

File: src/room.h [hunk 1]
```diff
@@ -214,7 +214,8 @@ typedef struct room_content_s
 
     float                       ambient_lighting[3];
     struct base_mesh_s         *mesh;                                           // room's base mesh
-    struct physics_object_s    *physics_body;
+    struct physics_object_s    *physics_body;                                   // static physics data
+    struct physics_object_s    *physics_body_dynamic;                           // static physics data
```
Comment (TeslaRus) [inline src/room.h]: fix comment, may be rename field (not good name)

**Comment by vvs-** [comment]
It fixes _Angkor Wat_. I have found no regressions. But #223 is still valid of course.

**Comment by TeslaRus** [comment]
#223 will be fixed / trying to fix in other branch, unfortunately I have no ideas how to fix it now...
that branch is only dynamic walls collision generation for rooms flipping (serious deal);

**Comment by vvs-** [comment]
> unfortunately I have no ideas how to fix it now...
I'm not sure, but I think that once I've seen a bug where Lara was able to pass a collisional portal in one direction, but not in its opposite. I think this was some time during altroom_refactoring in _Tomb of Qualopec_. If I get more time, I'll try to bisect it.

**Comment by vvs-** [comment]
Yes, I was right. This happened in 19d6085899e8c5d723eb352be7d4bb4031f974e7. But it was a special case when Lara climbed up from water. Still it could be an inspiration for a solution.

**Pull request merged**